torch_xla/csrc/tensor_util.cpp

```diff
@@ -746,30 +813,59 @@ bool TensorCompare(const at::Tensor& t1, const at::Tensor& t2) {
 }
 
 xla::ComputationClient::DataPtr TensorToXlaData(const at::Tensor& tensor,
-                                                const Device& device) {
-  return TensorToXlaData(
-      tensor, CreateComputationShapeFromTensor(tensor, &device), device);
+                                                const Device& device,
```

Comment:
I suggest we add metrics measurements to compute the latency of this code. Wdyt?

Reply:
sure, this function currently will never be called with  `transfer_async=True` because we only do that for seed upload. We can add a counter for future use through.

Reply:
Thanks.

Reply:
oh wait, this function is just a mask and will call the `TensorToXlaData` with a different signature. We already have metrics there